### PR TITLE
Ra 536 prototype

### PR DESCRIPTION
**Pull request by ingeknudsen**

### CODE REVIEW

File: src/components/job-overview/index.js [hunk 1]
```diff
@@ -169,7 +171,7 @@ JobOverview.propTypes = {
   appName: PropTypes.string.isRequired,
   job: PropTypes.exact(jobModel),
   jobName: PropTypes.string.isRequired,
-  repo: PropTypes.string,
+  repo: PropTypes.string.isRequired,
```
Comment (keaaa) [inline src/components/job-overview/index.js]: should this property be required? - there is a test in CommitHash component if repo property exist

File: src/components/jobs-list/job-summary.js [hunk 2]
```diff
@@ -71,8 +71,14 @@ const JobSummary = ({ appName, job }) => {
               <FontAwesomeIcon icon={faClock} size="lg" />
             </div>
             <div className="job-summary__data-list">
-              <RelativeToNow time={job.started} titlePrefix="Start time" />
-              <Duration title="Duration" start={job.started} end={job.ended} />
+              <RelativeToNow time={job.created} titlePrefix="Created time" />
```
Comment (keaaa) [inline src/components/jobs-list/job-summary.js]: when jobs now can be queued, should we show both Created and Started time?

Comment (nemzes) [reply src/components/jobs-list/job-summary.js]: I don't think the extra visual noise of an extra column is justified for something that has so few use cases… but we should display the relative time from start (as originally), not creation. Maybe something like this:
```jsx
{
  !job.started && (
  <React.Fragment>
    ⌛️ <RelativeToNow time={job.created} titlePrefix="Creation time" />
  </React.Fragment>
)}
{
  job.started && (
  <RelativeToNow time={job.started} titlePrefix="Start time" />
)}
```

File: src/components/job-overview/index.js (hunk 1 above)
Comment (nemzes) [reply src/components/job-overview/index.js]: Agreed; this shouldn't be required

**Pull request merged**